src/v3/src/components/Widget/index.tsx

```diff
@@ -463,8 +454,12 @@ export const Widget: FunctionComponent<WidgetProps> = (widgetProps) => {
                 fontWeight: 'bold',
                 wordBreak: 'break-all',
               },
+              '.no-translate': {
+                whiteSpace: 'nowrap',
```

Comment:
were elements with `no-translate` class not inheriting this style?

Reply:
I don't think so. It was added to the ScopedCSSBaseline to make sure it's applied everywhere in the SIW

cc @shawyu-okta

Reply:
I wonder if the styles in `GlobalStyles` should move into the `ScopedCssBaseline`.

Reply:
I'd also like to figure out how we can eliminate these two rules from the CSS baseline since they define some implicit behavior on all components, but that's not for this PR

Reply:
Moved these to `style.css` because they **were** being overridden by the stylis RTL plugin output specificity. Discussed with @shawyu-okta.